lib/json_schema_form/umm_preview_elements/umm_preview_science_keyword.rb

```diff
@@ -0,0 +1,18 @@
+# Provides the correct keyword levels to render a science keyword string
+
+# :nodoc:
+class UmmPreviewScienceKeyword < UmmPreviewKeyword
+  KEYWORD_LEVELS = %w(
+    Category
+    Topic
+    Term
+    VariableLevel1
+    VariableLevel2
+    VariableLevel3
+    DetailedVariable
+  ).freeze
+
+  # def initialize(data)
```

Comment:
should this be commented out? if so, remove it. Also, `umm_preview_service_keyword.rb` has it in, should both of them be removed? I think if the only thing they are doing is calling `super`, the answer is remove both